inference/core/interfaces/http/http_api.py

```diff
@@ -1617,6 +1630,48 @@ async def cog_vlm(
                         trackUsage(cog_model_id, actor)
                     return response
 
+            if CORE_MODEL_TROCR_ENABLED:
+
+                @app.post(
+                    "/trocr/ocr",
```

Comment:
should not that be `/ocr/trocr`

Reply:
Yes, you're correct. Side question: Should a DocTR endpoint with the new ontology? (if yes, we can set up a redirect, additional endpoint, etc)